ci-operator/jobs/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main-periodics.yaml

```
@@ -0,0 +1,88 @@
+periodics:
+- agent: kubernetes
+  cluster: build05
+  cron: 0 6 * * 1
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+    timeout: 4h0m0s
+  extra_refs:
+  - base_ref: main
+    org: Apicurio
+    repo: apicurio-registry-system-tests
+  labels:
+    ci-operator.openshift.io/cloud: aws
+    ci-operator.openshift.io/cloud-cluster-profile: aws-cspi-qe
+    ci-operator.openshift.io/variant: service-registry-ocp4.13-lp-interop
+    ci.openshift.io/generator: prowgen
+    job-release: "4.13"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: periodic-ci-Apicurio-apicurio-registry-system-tests-main-service-registry-ocp4.13-lp-interop-service-registry-2-3
```

```suggestion
  name: periodic-ci-Apicurio-apicurio-registry-system-tests-main-service-registry-ocp4.13-lp-interop-service-registry-2-3
  prowjob_defaults:
    tenant_id: gangway-api
```

Please add this here as this will allow for us to trigger this job outside of the cron if needed.

---

If you add the line @ascerra [mentioned earlier](https://github.com/openshift/release/pull/38858/files#r1196983286) and run `make update`, this will be done for you.